package.json

```diff
@@ -82,10 +82,10 @@
   "dependencies": {
     "classlist-polyfill": "^1.0.3",
     "dom-delegate": "^2.0.3",
-    "jquery": "~1.11.0",
-    "jquery.easing": "~1.3.0",
+    "jquery": "^3.3.1",
+    "jquery.easing": "^1.4.1",
```

Comment:
I agree that we should remove jquery.easing here before merging.

Reply:
Fixed in https://github.com/cfpb/capital-framework/pull/747/commits/9779d767e7fac5789666ecde170f829c5a694c73.